src/utils.ts

```diff
@@ -19,7 +19,7 @@ export function toWikiLink(input: string): string {
 }
 
 export function toKoldbLink(input: string): string {
-  return `https://www.koldb.com/player.php?name=${encodeURI(input)}`;
+  return `http://www.koldb.com/player.php?name=${encodeURI(input)}`;
```

Comment:
i respect oxbarn too much to call him out for this but it is pretty funny to me that https just straight does not work